tests/Doctrine/Tests/DBAL/SQLParserUtilsTest.php

```diff
@@ -356,6 +356,15 @@ public function dataExpandListParameters()
                 array(1, 2),
                 array(\PDO::PARAM_INT, \PDO::PARAM_INT)
             ),
+            // Test parameters that require a different binding type provided by Type::getBindingType()
+            array(
+                "SELECT * FROM Foo WHERE foo IN (?)",
+                array(array('oof', 'rab')),
+                array('customReverseType'),
```

Comment:
Excuse me but I still don't get the binding type here. We only have limited types that we map to PDO types, and this one is not supported... How should we map that one to the driver? Or am I missing something here?